kuma/static/js/analytics.js

```diff
@@ -41,6 +41,10 @@
             if(ga && ga.create) {
                 // Send event to GA
                 ga('send', data);
+                // execute callback now
+                if(callback) {
```

Comment:
given that all branches of the outer conditional call the callback, I'd recommend reverting this, removing L56-58 and removing the else from 60